org-code-javabuilder/lib/src/main/java/org/code/neighborhood/Helpers.java

```diff
@@ -0,0 +1,16 @@
+package org.code.neighborhood;
+
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Helpers {
+    private static final String HEX_WEB_COLOR_PATTERN
+            = "^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$";
+    private static final Pattern PATTERN = Pattern.compile(HEX_WEB_COLOR_PATTERN);
+    private static final Set<String> WEB_COLORS = Set.of("white", "silver", "gray", "black", "red", "maroon", "yellow", "olive", "lime", "green", "aqua", "teal", "blue", "navy", "fuchsia", "purple");
```

Comment:
should we add a comment/todo here to add the 'extended' colors or are we sticking with the basic colors?

Reply:
I'll do this as a follow up!